clang/lib/Driver/ToolChains/Clang.cpp

```diff
@@ -9486,17 +9513,30 @@ void OffloadBundler::ConstructJobMultipleOutputs(
     // FPGA device triples are 'transformed' for the bundler when creating
     // aocx or aocr type bundles.  Also, we only do a specific target
     // unbundling, skipping the host side or device side.
-    if (types::isFPGA(InputType)) {
+    if (types::isFPGA(InputType) || InputType == types::TY_Tempfilelist) {
       if (getToolChain().getTriple().isSPIR()) {
         if (Dep.DependentToolChain->getTriple().getSubArch() ==
             llvm::Triple::SPIRSubArch_fpga) {
+          // mtoguchi
```

Comment:
What is this comment for?

Reply:
oops, I use my idsid as a marker when editing :)